var/spack/repos/builtin/packages/cosmomc/package.py

```diff
@@ -0,0 +1,200 @@
+
+        # Set up Planck data if requested
+        clikdir = join_path('data', 'clik')
+        try:
+            os.remove(clikdir)
+        except OSError:
+            pass
+        if '+planck' in spec:
+            os.symlink(join_path(os.environ['CLIK_DATA'], 'plc_2.0'), clikdir)
+        else:
+            os.environ.pop('CLIK_DATA', '')
+            os.environ.pop('CLIK_PATH', '')
+            os.environ.pop('CLIK_PLUGIN', '')
+
+        # Choose compiler
+        # Note: Instead of checking the compiler vendor, we should
+        # rewrite the Makefile to use Spack's options all the time
+        if spec.compiler.name == 'gcc':
+            if not spec.compiler.version.satisfies(ver('6:')):
+                raise InstallError(
+                    "When using GCC, "
+                    "CosmoMC requires version gcc@6: for building")
+            choosecomp = 'ifortErr=1'       # choose gfortran
+        elif spec.compiler.name == 'intel':
+            if not spec.compiler.version.satisfies(ver('14:')):
+                raise InstallError(
+                    "When using the Intel compiler, "
+                    "CosmoMC requires version intel@14: for building")
+            choosecomp = 'ifortErr=0'       # choose ifort
+        else:
+            raise InstallError("Only GCC and Intel compilers are supported")
+
+        # Configure MPI
+        if '+mpi' in spec:
+            wantmpi = 'BUILD=MPI'
+            mpif90 = 'MPIF90C=%s' % spec['mpi'].mpifc
+        else:
+            wantmpi = 'BUILD=NOMPI'
+            mpif90 = 'MPIF90C='
+        
+        # Choose BLAS and LAPACK
+        lapack = ("LAPACKL=%s" %
+                  (spec['lapack'].lapack_libs +
+                   spec['blas'].blas_libs).ld_flags)
+
+        # Build
+        make(choosecomp, wantmpi, mpif90, lapack)
+
+        # Install
+        mkdirp(prefix.bin)
+        install('cosmomc', prefix.bin)
+        root = join_path(prefix.share, 'cosmomc')
+        mkdirp(root)
+        for entry in ['batch1',
+                      'batch2',
+                      'batch3',
+                      'camb',
+                      'chains',
+                      'clik_latex.paramnames',
+                      'clik_units.paramnames',
+                      'cosmomc.cbp',
+                      'data',
+                      'distgeneric.ini',
+                      'distparams.ini',
+                      'disttest.ini',
+                      'docs',
+                      'job_script',
+                      'job_script_MOAB',
+                      'job_script_SLURM',
+                      'paramnames',
+                      'params_generic.ini',
+                      'planck_covmats',
+                      'python',
+                      'scripts',
+                      # don't copy 'source'
+                      'test.ini',
+                      'test_pico.ini',
+                      'test_planck.ini',
+                      'tests',
+        ]:
```

Comment:
This line is causing the flake8 tests to fail. I would just move this bracket right after `'tests'`, or create a list and the run `for entry in file_list:` or something.